src/grossToNet.js

```diff
@@ -1,10 +1,10 @@
-const Big = require('big.js');
+import Big from 'big.js';
 const todayDate = new Date();
-const calculateTaxes = require('./taxCalculator.js');
-const calcInsuranceDetails = require('./insurancesCalculator.js');
-const {SELF_DEDUCTION, DEDUCTION_PER_PERSON} = require("./salaryConstants");
+import {calculateTaxes} from './taxCalculator';
+import calcInsuranceDetails from './insurancesCalculator';
+import {SELF_DEDUCTION, DEDUCTION_PER_PERSON} from './salaryConstants';
 
-module.exports = function net(gross, dependents = 0, region = 1, date = todayDate) {
+export default function grossToNet(gross, dependents = 0, region = 1, date = todayDate) {
```

Comment:
Since this file name has been changed, you have to update all unit tests that import the old one "net.js"